content/backend/graphql-js/5-connecting-server-and-database.md

```diff
@@ -272,6 +270,23 @@ The Prisma CLI also uses information that's provided in `.graphqlconfig.yml`. Th
 
 <Instruction>
 
+Update `prisma.yml` to include a deploy hook:
+
+```bash(path=".../hackernews-node/database/prisma.yml")
+endpoint: `https://eu1.prisma.sh/public-graytracker-771/hackernews-node/dev
+datamodel: datamodel.graphql
+
+# Deploy hook
+hooks:
+  post-deploy:
+    - graphql get-schema --project database
```

Comment:
I think we're missing instructions in the tutorial to install the GraphQL CLI, so as of now the invokation of the hook is likely to fail if the reader doesn't have the CLI installed globally